WooCommerce/Classes/ViewRelated/Orders/Simple Payments/Method/SimplePaymentsMethodsViewModel.swift

```diff
@@ -60,10 +62,6 @@ final class SimplePaymentsMethodsViewModel: ObservableObject {
     ///
     private let presentNoticeSubject: PassthroughSubject<SimplePaymentsNotice, Never>
 
-    /// Observes the store's current CPP state.
-    ///
-    private let cppStoreStateObserver: CardPresentPaymentsOnboardingUseCaseProtocol
```

Comment:
We should probably avoid changing how this screen behaves, pending confirmation from the ongoing design work...

Reply:
Indeed, reverted in https://github.com/woocommerce/woocommerce-ios/pull/6805/commits/44aec37c39e357cede54dfdabf664ff99de737ff